### DISCUSSION TITLE
GDP and Population Data for Africa

### DISCUSSION THREAD

**Discussion by pz-max** [pypsa-meets-earth/pypsa-earth#46] [General]
Let's share here some data sources on GDP and population for Africa. 
We plan to fill GADM zones with GDP and population data.

**Comment by pz-max** [comment]
@davide-f

**Comment by pz-max** [comment]
Population data:
- Explanation between bottom-up and top-down data at [WorldPop](https://www.worldpop.org/methods/populations)
- WorldPop offers [open data](https://www.worldpop.org/geodata/summary?id=139) using [peanutButter method](https://github.com/wpgp/peanutButter)
- WorldPop is also used at the World Ressource Institute (WRI) for the [Energy Access Explorer](https://www.energyaccessexplorer.org/tool)
- Nasa offers a couple of rasterized global [population datasets](https://sedac.ciesin.columbia.edu/data/collection/gpw-v4/sets/browse)

Global population & GDP. 
- Used in GlobalEnergyGIS. Original data: http://www.cger.nies.go.jp/gcp/population-and-gdp.html. Raster converted: https://github.com/Nowosad/global_population_and_gdp
- Use [National GDP](https://data.worldbank.org/indicator/ny.gdp.mktp.cd) data. We could disaggregate the national GDP with our population raster/ population density raster. A non-linear distribution/relation between GDP and population density could also reflect that regions around cities often contribute more to the GDP (+ probably have higher energy demand/access per capita)
- Leverage existing research. Kummu, M., Taka, M. & Guillaume, J. Gridded global datasets for Gross Domestic Product and Human Development Index over 1990–2015. Sci Data 5, 180004 (2018). https://doi.org/10.1038/sdata.2018.4

**Comment by davide-f** [comment]
I investigated a number of datasets; primarily NASA and WorldPop are the most easy to use probably.
I decided to use the WorldPop, as also you suggested, and it works fine. It allows to download data by country, which is easier to match with the GADM dataset because we can compare each country a time; instead in the NASA one, the data are downloaded by squares which makes the above more tricky: we should merge all the images which is unconfortable from a computational point of view.

I decided to use the dataset 100m resolution 2020 dataset UN-adjusted. With respect to the non-adjusted version, the UNadj aims at scaling the previous dataset to match the population estimates by UN.
If you don't agree, we can use the other dataset, by simply changing a string.

I successfully developed the script to automatically download the data.

**Reply by pz-max** [reply]
Cool! The worldpop data gives you the country data as a raster?

**Reply by davide-f** [reply]
Yes

**Comment by davide-f** [comment]
I believe that the problem on the population data is solved; we can develop and refine the methodology for identifying the population, but the next element is the GDP.

I foresee the following options:
a) (preferred) if a raster dataset of GDP is available, we could proceed similarly to the population data; however, I feat that's quite unlikely. Some datasets exist, but they are mostly country-level. Anyway, I'll investigate this and publish some options
b) (worst case) if we assume that GDP is linearly correlated to population, we can scale down the national GDP (available) to the shapes by using the population layer. This is clearly the worst case, as the correlation is not exactly linear

**Reply by pz-max** [reply]
In addition. You could use instead of the population the population density = population in GADM zone/Area of GADM zone

This might slightly better reflect the observed reality. It would mean higher GDP/capita in dense populated regions compared to less dense regions.

**Reply by euronion** [reply]
Maybe this source avoid reinventing the wheel?

Kummu, M., Taka, M. & Guillaume, J. Gridded global datasets for Gross Domestic Product and Human Development Index over 1990–2015. Sci Data 5, 180004 (2018). https://doi.org/10.1038/sdata.2018.4
